skfda/exploratory/visualization/representation.py

```diff
@@ -79,249 +113,395 @@ def _get_color_info(fdata, group, group_names, group_colors, legend, kwargs):
     return sample_colors, patches
 
 
-def plot_graph(fdata, chart=None, *, fig=None, axes=None,
-               n_rows=None, n_cols=None, n_points=None,
-               domain_range=None,
-               group=None, group_colors=None, group_names=None,
-               legend: bool = False,
-               **kwargs):
-    """Plot the FDatGrid object graph as hypersurfaces.
+class GraphPlot:
+    """
+    Class used to plot the FDataGrid object graph as hypersurfaces.
 
-    Plots each coordinate separately. If the :term:`domain` is one dimensional,
-    the plots will be curves, and if it is two dimensional, they will be
-    surfaces.
+    A list of variables (probably depths) can be used as an argument to
+    display the functions wtih a gradient of colors.
```

Comment:
I would remove this and explain it in the arguments OR explain the different ways in which color can be specified. Also gradients are not only useful for depths, could be useful also for scalar regresion targets, for example.